metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -19,28 +19,27 @@ namespace annot {
 
 using namespace mtg::annot::binmat;
 using mtg::common::logger;
+/** Marker type to indicate a value represent a node index in a BOSS graph */
+using node_index = graph::boss::BOSS::node_index;
```

Comment:
not used
```suggestion
```

Reply:
Lines 119, 121.

Reply:
I don't see it there

Reply:
``` 
 uint64_t node_index = graph.boss_to_kmer_index(pred);
pred_buf[r].push_back(
      graph::AnnotatedSequenceGraph::graph_to_anno_index(node_index));
```

Reply:
so it doesn't use `graph::boss::BOSS::node_index`, and even if it did, it would have to be `DBGSuccinct::node_index`, not `BOSS::node_index`

Reply:
`node_index` is a variable name here, not the type.

Reply:
Duh, right.